standard/navbar.lua

```diff
@@ -1,122 +1,156 @@
-local p = {}
+-- Original source copied from Wikipedia on 2017-12-20.
+-- Modified (code style + using Class Module) on 2021-12-?? for liquipedia
 
-local getArgs
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
 
-function p._navbar(args)
-	local titleArg = 1
+local NavBar = {}
 
-	if args.collapsible then
-		titleArg = 2
-		if not args.plain then
-			args.mini = 1
-		end
-		if args.fontcolor then
-			args.fontstyle = 'color:' .. args.fontcolor .. ';'
-		end
-		args.style = 'float:left; text-align:left'
-	end
+--legacy entry points
+function NavBar.navbar(args, args1, arg2)
+	return NavBar.NavBar(args, args1, arg2)
+end
+function NavBar._navbar(args, args1, arg2)
+	return NavBar.NavBar(args, args1, arg2)
+end
 
-	local titleText = args[titleArg] or (':' .. mw.getCurrentFrame():getParent():getTitle())
-	local title = mw.title.new(mw.text.trim(titleText), 'Template');
+local _BRACKETS_LEFT = '&#91;'
+local _BRACKETS_RIGHT = '&#93;'
+
+function NavBar.NavBar(args, arg1, arg2)
```

Comment:
If there's a way to clear up the variable names, that would be great

Reply:
not without having to adjust it everywhere